examples/c++/CMakeLists.txt

```diff
@@ -103,8 +103,11 @@ add_example (slide)
 add_example (groveslide)
 add_example (buzzer-sound)
 add_example (nrf24l01-transmitter)
+set_source_files_properties(${example_src} PROPERTIES COMPILE_FLAGS -Wno-tautological-compare)
 add_example (nrf24l01-receiver)
+set_source_files_properties(${example_src} PROPERTIES COMPILE_FLAGS -Wno-tautological-compare)
 add_example (nrf24l01-broadcast)
+set_source_files_properties(${example_src} PROPERTIES COMPILE_FLAGS -Wno-tautological-compare)
```

Comment:
This is not the right way to enable compiler flags see how mraa now does it - https://github.com/intel-iot-devkit/mraa/blob/master/CMakeLists.txt#L13